templates/cowcheck/0/rancher-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '2'
+catalog:
+  name: cowcheck
+  version: 0.2.0
+  description: A microservice for checking the health of a Rancher node
+  minimum_rancher_version: v.1.6.0
+  questions:
+    - variable: POLL_INTERVAL
+      label: POLL_INTERVAL
+      description: Frequency in seconds to run checks
+      type: string
+      default: 1.24
+    - variable: ENABLE_STORAGE_CHECK
+      label: ENABLE_STORAGE_CHECK
+      description: Enable storage check
+      type: boolean
+      default: false
+    - variable: DATA_SPACE_THRESHOLD
+      label: DATA_SPACE_THRESHOLD
+      description: Minimum amount of free Docker Data Storage space in bytes
+      type: string
+      default: 1.24
```

Comment:
If this value is the minimum amount of free bytes, may be better a default value of 1024, 10240...??
```
-      default: 1.24
+      default: 1024
```